ui/src/components/date/QasDate.vue

```diff
@@ -94,15 +94,16 @@ export default {
 
       return {
         class: this.classes,
-        color: 'white',
+        color: 'primary',
         events: this.normalizedEvents,
         mask: this.mask,
         minimal: true,
         multiple: this.multiple,
         options: this.normalizedOptions,
         ref: 'date',
         style: this.styles,
-        textColor: 'primary',
+        textColor: 'white',
+        eventColor: date => this.getEventColor(date),
```

Comment:
somente `eventColor: this.getEventColor` não funfa?

Reply:
feito